packages/react-components/lib/themes/global-theme.tsx

```diff
@@ -2,22 +2,24 @@ import { withStyles } from '@material-ui/core/styles';
 
 export const GlobalCss = withStyles((theme) => ({
   '@global': {
+    // override accordian background to distinguish it from onipanel
     '.MuiAccordion-root': {
       backgroundColor: theme.palette.background.default,
     },
+    // override text color of disabled button to make it visible
```

Comment:
I don't really agree with this change. Disabled text have their own color so that they look disabled.